OpenTabletDriver.UX/App.cs

```diff
@@ -83,7 +84,7 @@ public Settings Settings
             ProgramDescription = "Open source, cross-platform tablet configurator",
             WebsiteLabel = "OpenTabletDriver GitHub Repository",
             Website = new Uri(@"https://github.com/OpenTabletDriver/OpenTabletDriver"),
-            Version = $"v{Version}",
+            Version = $"v{Version}, Built on {BuildDate}",
```

Comment:
```suggestion
            Version = $"v{Version}, built on {BuildDate}",
```
But this in itself is a weird change for me, build date is not part of a version. The date that a software is built does not indicate the version, nor change its behaviour. We can have the same "version" of a software built on different dates.

This should be moved out to another entry in diagnostics, and removed from UX.

Reply:
> This should be moved out to another entry in diagnostics, and removed from UX.

You're right, as we've added the git hash to the version we don't really need to have the build date in the UX, just the exported diag. I'll do that.